templates/circa/templates.yaml

```diff
@@ -0,0 +1,55 @@
+dataset: circa
+templates:
+  053260a8-1bcc-4805-81d2-bb528fc56ca2: !Template
+    id: 053260a8-1bcc-4805-81d2-bb528fc56ca2
+    jinja: 'Convert the question to declarative form:
+
+
+      {{question_X}} |||
+
+      {{canquestion_X}}'
+    name: question_declarative
+    reference: ''
+  70b7a94a-6a39-4a81-9a6e-0709a0acdb28: !Template
+    id: 70b7a94a-6a39-4a81-9a6e-0709a0acdb28
+    jinja: "What is the most common judgement in a less strict way given the question-answer\
+      \ pair of X and Y in the context of {{context}}?\n\nX: {{question_X}} \nY: {{answer_Y}}\
+      \ |||\n\n{{   \n [\n  \"Yes\",\n  \"No\",\n  \"In the middle, neither yes nor\
+      \ no\",\n  \"Yes, subject to some conditions\",\n  \"Other\"\n]\n [goldstandard2]\
```

Comment:
Wrap the whole template into an if condition: `{% if goldstandard2 != -1 %} template {% endif %} otherwise there will be errors introduced when the label is missing